lab1.js

```diff
@@ -72,11 +74,24 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  HINT: the 'split' method on String will be useful.
 */
 
-//your code goes here
+function chirps (sentence) {
+	var output = "chirp";
+	for (var i = 0; i < sentence.length; i++) {
+		if (sentence[ i ] === " ") {
+			output += " chirp";
+		}
+	}
+	output += ".";
+	console.log(output);
+	return output;
+}
+
+sentence1 = chirps(sentence1);
+sentence2 = chirps(sentence2);
 
```

Comment:
forgot to use another kind of loop here. No points off for now, but in the future make sure to use all the stuff I specify in the question.